Doc/library/stdtypes.rst

```diff
@@ -2191,6 +2484,19 @@ expression support in the :mod:`re` module).
    test string beginning at that position.  With optional *end*, stop comparing
    string at that position.
 
+   For example:
+
+      >>> 'Python'.startswith('Py')
+      True
+      >>> 'a tuple of prefixes'.startswith(('at', 'in'))
+      False
+      >>> 'a tuple of suffixes'.startswith(('at', 'a'))
+      True
+      >>> 'Python is amazing'.startswith('is', 7)
+      True
+
+   See also :meth:`endswith`.
```

Comment:
As above, could be useful to see also `removeprefix`

Reply:
Done.